weatherman/weatherman.py

```diff
@@ -0,0 +1,78 @@
+import csv
+import os
+import pandas
+import glob
+from statistics import mean
+
+class WeatherReading:
+	def __init__(self,row):
+		self.date=row['PKT']
+		self.max_temp=row['Max TemperatureC']
+		self.min_temp=row['Min TemperatureC']
+		self.mean_hum=row[' Mean Humidity']
+		self.max_hum=row['Max Humidity']
+
+	def __str__(self):
+		return f'{self.date}, {self.max_temp}, {self.min_temp}, {self.mean_hum}, {self.max_hum}'
+
+	def write_dict(self):
+		weather_dict={'date' : self.date, 'max_temp' : self.max_temp, 'min_temp' : self.min_temp, 'mean_hum' : self.mean_hum, 'max_hum' : self.max_hum}
+		return weather_dict
+	
+
+class Reader:
+	_weather_data = []		
+	def read_files(self,opt,path,year,month="*"):
```

Comment:
change it to `__init__` and return the output to be used in pipelines

Reply:
with __init__ i cannot return the list, as it can only return a string through __str__